apps/web/README.md

```
@@ -1,36 +1,402 @@
+# ルートディレクトリから
+pnpm --filter web dev
+
+# またはwebディレクトリで直接
+cd apps/web
 pnpm dev
-# or
-bun dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+ブラウザで http://localhost:3000 を開いて確認できます。
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)